src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,62 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { FilterCentury } from './filterCentury/FilterCentury';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [search, setSearch] = useState('');
```

Comment:
i also think that we can rewrite it without using `useState` here (cause we have `useSearchParams`)